native/cocos/bindings/auto/jsb_render_auto.h

```diff
@@ -35,8 +35,10 @@ SE_DECLARE_FUNC(js_render_PipelineRuntime_destroy);
 SE_DECLARE_FUNC(js_render_PipelineRuntime_getMacroBool);
 SE_DECLARE_FUNC(js_render_PipelineRuntime_getMacroInt);
 SE_DECLARE_FUNC(js_render_PipelineRuntime_getMacroString);
+SE_DECLARE_FUNC(js_render_PipelineRuntime_isResetRenderQueue);
 SE_DECLARE_FUNC(js_render_PipelineRuntime_onGlobalPipelineStateChanged);
 SE_DECLARE_FUNC(js_render_PipelineRuntime_render);
+SE_DECLARE_FUNC(js_render_PipelineRuntime_resetRenderQueue);
```

Comment:
Don't have to commit modifications of auto generated codes, only new files are needed(jsb_xr_auto.h and jsb_xr_auto.cpp). These files are updated automatically by CI.